app/main.py

```
@@ -1 +1,17 @@
-# write your code here
+def copy_file(command: str) -> None:
+    splitted_entry_command = command.strip().split()
+
+    if len(splitted_entry_command) != 3 or splitted_entry_command[0] != "cp":
+        return
+
+    file_to_copy = splitted_entry_command[1]
+    new_file = splitted_entry_command[2]
+
+    if file_to_copy == new_file:
+        return
+
+    try:
+        with open(file_to_copy, "r") as f_in, open(new_file, "w") as f_out:
+            f_out.write(f_in.read())
+    except FileNotFoundError:
+        return
```

The function catches `FileNotFoundError` and silently returns. The task description does not require adding extra error-handling behavior beyond validating the command structure; avoid inventing additional behaviors (the description defines only the required command validation and copy/no-op behaviour). Consider removing this try/except so the function only performs the required validation and copying, or explicitly document why this additional silent handling is necessary. See the task description for expected behavior.